blurred background and removal of border for video and slides

shubham-tyagi opened the pull request:
Signed-off-by: shubham-tyagi <raj.tyagi2000@gmail.com>

**Description**
As discussed in the video, the background is blurred and the border for the modal is removed. I have made some really small modifications apart from that.

![](https://imgur.com/zOicNNE.png)

This PR fixes #

**Notes for Reviewers**
Please point out if I missed something or if anything else is needed to be done. I could remove those small modifications if they are not looking good.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

<!--
Thank you for contributing to Meshery! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

debo19 commented on collections/_pages/talks-and-presentations.html:
```diff
@@ -57,14 +57,15 @@
   margin: 0 2rem 0 2rem;
-}
-.modal-window > div {
   border-radius: 1rem;
```
I don't think we should apply curved borders to the video.

vedant-kakde replied:
+1

shubham-tyagi replied:
okay will revert that, does it not look good or some design violations?

debo19 replied:
@shubham-tyagi no not any violation, just looks a little blunt to me, you can try reducing the border-radius a bit  ( 0.5 rem maybe ), that should work too.

shubham-tyagi replied:
yeah sure, will do that.

debo19 reviewed (commented):
Other changes look good.

debo19 reviewed (commented):
This button should be center-aligned.

![button](https://user-images.githubusercontent.com/85789734/139922760-4b3a193d-1192-487f-b00a-f75670a5ea9a.jpg)

shubham-tyagi commented:
Hey @debo19 , I will do the button center aligned , for now can you review my recent commit. 

- I have removed the image link so that even if we click on the image it font gets enlarged
- I have also added a functionality that if we click outside of the overlay(not only on the cross) then too the overlay will vanish.

debo19 commented:
@shubham-tyagi The button isn't working for me in the preview, the modal isn't visible. Is it working fine for you?

shubham-tyagi commented:
@debo19 In my local machine, it's working fine. What might be the problem here?

shubham-tyagi commented:
I guess Its not working because the JS file is not included in the preview(I am just guessing). Because in my local machine as you can see in the video it's working fine. @warunicorn19 thoughts here ?

![ezgif com-gif-maker](https://user-images.githubusercontent.com/41378464/139941522-079a88d5-f63e-4e76-a856-c25d61084dcf.gif)

shubham-tyagi commented:
Hey @debo19 , It's working fine in the preview now. Please see if its working fine in your machine.

warunicorn19 reviewed (changes requested):
@shubham-tyagi really like the javascript approach😅, couple of nitpicks, could you please center align the button and the <script> tag that you mentioned at the end of the page, could you please put that right below the <style> tag (good practice to maintain that format).

Good work💪.

shubham-tyagi commented:
@warunicorn19 when I include script tag just after style tag then it reverses the page. And I am not sure but isn't it best to include js script after the page has been loaded? What do you think should be done @debo19 @warunicorn19 

![](https://imgur.com/kWTrvBn.png)

warunicorn19 commented:
huh🤔? i guess what you're saying makes sense because html loads from top to bottom. okay, ignore what i said about the <script> tag. aligning the button to the center will do

shubham-tyagi commented:
@warunicorn19  Could you please check out the screenshot I attached in my previous comment, Should it full-page centered or this is fine?

warunicorn19 commented:
Yup the position in the screenshot works👌

debo19 commented on collections/_pages/talks-and-presentations.html:
```diff
@@ -102,34 +101,33 @@
-        <a href="#open-modal" class="link">Find video and slides here</a>  
-      </div>
+        <div class="button-para" style="margin-top: 3rem; ">
+          <button class="link" style="position: relative;left: 25%;">Find video and slides here</button>  
```
![mobile2](https://user-images.githubusercontent.com/85789734/140069553-c37895be-cb3b-4b06-b7fb-880d6a31f904.jpg)
The button should be aligned to the center in mobile view too.  Also, can you remove the inline styles and add them inside the style tag.

debo19 reviewed (commented):
Great work here @shubham-tyagi 🚀 . This should be the last change 😅

warunicorn19 reviewed (approved):
LGTM! Thanks @shubham-tyagi
